app/Models/Order.php

```diff
@@ -11,7 +11,7 @@
 class Order extends Model
 {
     //attributes id, date, status, total, user, created_at, updated_at
-    protected $fillable = ['date', 'status',  'total', 'user_id',];
+    protected $fillable = ['date', 'status', 'total', 'user_id', 'credit_card_id'];
```

Comment:
Pipe, please check the order of this array, arrays must be in alphabetical order.